dronegen/mac.go

```diff
@@ -277,28 +349,78 @@ func installNodeToolchainStep(workspacePath string) step {
 	}
 }
 
-func cleanUpToolchainsStep(path string) step {
-	return step{
+func configureToolchainsCommands(config toolchainConfig) []string {
+	commands := []string{
+		// HOME needs to be set to the actual user rather than what Drone sets it to by default. This
+		// way we're able to unlock Keychain which is needed for Connect signing.
+		`echo HOME=$${HOME}`,
```

Comment:
Is this `echo` needed for some reason, or it's here just for debug?

Reply:
I think it might be a leftover from #12257 when Trent was trying to make signing work (688a3712b). It doesn't seem to serve any purpose anymore as it just shows what Drone has set HOME to and the var gets immediately overwritten by the next command. I'll remove it.